docs/source/install.rst

```diff
@@ -1,4 +1,4 @@
-Installation Guide
+Installation
 ==================
```

Comment:
Nitpick: make this line equally long as the title above
```suggestion
============
```

Reply:
:+1: